app/routes/events/view/sessions/list.js

```diff
@@ -1,6 +1,7 @@
 import Route from '@ember/routing/route';
-
-export default Route.extend({
+import { action } from '@ember/object';
```

Comment:
@CosmicCoder96 actually I'm already refactoring this file in my PR #3198 where I'm also introducing few session rating columns and actions. :sweat_smile: 
I'll introduce any changes if missing there.